test/target_mvd/targets_test.exs

```diff
@@ -108,8 +113,11 @@ defmodule TargetMvd.TargetsTest do
              }
     end
 
-    test "create_target/1 with invalid data returns error changeset" do
-      assert {:error, %Ecto.Changeset{}} = Targets.create_target(@invalid_attrs)
+    test "create_target/1 with invalid data returns error changeset", %{topic: topic, user: user} do
+      assert {:error, %Ecto.Changeset{}} =
+               Targets.create_target(
+                 Map.merge(@invalid_attrs, %{"topic_id" => topic.id, "user_id" => user.id})
```

Comment:
I'm not 100% sure, but isn't `%{ @invalid_attrs | "topic_id" => topic.id, "user_id" => user.id}` equivalent to `Map.merge`? @brunvez probably knows if this works or not 😅

Reply:
Im not sure about this, I think the `|` operator is for lists exclusively, but I'll look into it.

Reply:
You can use the `|` operator but only if the key exists I believe, it's just for updates